spyder/config/base.py

```diff
@@ -144,15 +149,15 @@ def get_conf_path(filename=None):
 
     # Create conf_dir
     if not osp.isdir(conf_dir):
-        if running_under_pytest():
+        if running_under_pytest() or CLEAN:
             os.makedirs(conf_dir)
         else:
             os.mkdir(conf_dir)
     if filename is None:
         return conf_dir
     else:
         return osp.join(conf_dir, filename)
-        

```

Comment:
Please avoid adding/removing blanks in the 3.x branch. It makes my life harder when merging with `master` and also with `split-plugins`. All pep8 stuff needs to be done in `master`, not here, because this branch is in maintenance now.

Reply:
Sorry; my mistake. It must have happened when I did the ``else:`` change; one more reason to avoid that sort of thing I suppose.